src/content/docs/experiment/onmonetization.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: onmonetization event handler
+---
+
+import { LinkOut, Hidden } from "@interledger/docs-design-system";
+import { Badge } from '@astrojs/starlight/components';
+import Specification from '/src/components/docs/Specification.astro'
+import BrowserCompat from '/src/components/docs/BrowserCompat.astro'
+import data from '/src/data/browser-compat-data/onmonetization.json'
+
+<Badge text="HTML DOM API" style="position: relative; top:-30px;" />
+
+By creating an `onmonetization` event handler, your web page can programmatically respond to `monetization` events.
+
+## `window.onmonetization` 
+
+The `onmonetization` event handler is used by the [Web Monetization agent](/resources/glossary/#web-monetization-agent) to process `monetization` events sent to the browser `window` (the event target). 
+
+The `onmonetization` property can be applied to any element and is used by the browser to notify that some `<link>` element is [monetized](/experiment/link-element). 
```

Comment:
Not exactly every element, but: `window`, any monetization link element and any element that has a child monetization link element (i.e. `head`, `body`, `audio` etc). Relevant: https://github.com/WICG/webmonetization/pull/458